client/asset/btc/btc.go

```diff
@@ -72,6 +72,10 @@ const (
 	redeemFeeBumpFee = "redeemfeebump"
 )
 
+const (
+	minTimeBeforeAcceleration uint64 = 3600 // 1 hour
+)
```

Comment:
The point of this was to limit the user from accelerating too frequently and too early. This is the question that @JoeGruffins and I were discussing: should we give the user full control, or should we include things like this to get them to avoid unnecessarily accelerating? What do you think?

Reply:
Ah.  I think I'll have a better feeling when testing on testnet or mainnet, but I suspect I'd feel a little frustrated if I saw a large fee spike and my tx getting buried under 5 blocks worth of mempool only to have my software tell me "too soon".
On the other hand, people will click accelerate when there is absolutely no use, e.g. their txns were already gonna be in the next block, but they felt the need to "accelerate" because they didn't check mempool.space or whatever.

Reply:
How about we put a warning if they are trying to accelerate, but it's very likely that their transaction will be included in the next block?

Reply:
As I pointed out, bitpay makes you wait four hours to accelerate according to their docs. I think an hour forced is not that long. Maybe fee goes down and gets mined anyway. Just my opinion, feel free to ignore.

Reply:
> very likely that their transaction

This could be tricky to characterize.  Certainly the fee rate and fee rate estimates don't tell the whole story because next block could be half full, or the tx could be 5 blocks worth of mempool deep but the "optimal" fee according to an oracle is only a little higher.

I was tempted to suggest in the desc/tooltip that the user should consult a mempool oracle to make an informed decision, because that's personally what I would do.

Reply:
I'm thinking to still check if the acceleration is too soon, and instead of completely blocking them, have a second popup prompting them to check the mempool oracle to make sure they want to accelerate.